### ISSUE TITLE
Deploy-EFlow failed with L6287

### ISSUE THREAD

**Issue by JohnMasen** [Azure/iotedge-eflow#272]
**Describe the bug**
While trying to deploy iotedge with Deploy-EFlow, the process failed with error code L6287

**To Reproduce**
1. Download and install MSI package from https://aka.ms/AzEFLOWMSI_1_4_LTS_X64
2. Open powershell as Admin
3. Run command "Deploy-EFlow"
4. install failed at Step1 with error "- The system cannot find the file specified. (L6287)"

PS: I have uninstall/reinstalled iot edge MSI several times, but this issue keep happening.
**Expected behavior**
Install successful

**Screenshots**
```Console
PS C:\Program Files> Deploy-EFlow

[10/24/2023 23:18:00] Deploying Azure IoT Edge for Linux on Windows


[10/24/2023 23:18:02] Enabling Microsoft Update. This will allow Azure IoT Edge for Linux on Windows to receive updates.
 - Microsoft Update is enabled.

[10/24/2023 23:18:02] Step 1: Preparing host for Azure IoT Edge for Linux on Windows


[10/24/2023 23:18:02] Checking host for required features

 - Checking the status of 'Microsoft-Hyper-V'
 - Checking the status of 'Microsoft-Hyper-V-Management-PowerShell'
 - Checking the status of 'Microsoft-Hyper-V-Hypervisor'
 - Checking the status of 'OpenSSH.Client*'
 - Verifying whether Hyper-V is enabled and functional

[10/24/2023 23:18:08] Exception caught!!!

 - The system cannot find the file specified. (L6287)

[10/24/2023 23:18:08] In order to attempt another deployment, please uninstall the Azure IoT Edge msi and start from fresh

The system cannot find the file specified.
```

**Windows Host OS (please complete the following information):**
 - Edition: Windows 11 Pro
 - Version: 10.0.22621.2428
 - Virtual Machine: Not VM
 
**Additional context**
<img width="765" alt="image" src="https://github.com/Azure/iotedge-eflow/assets/7631912/1c586f87-8e46-42c8-b827-dff6c169b2e0">

**Comment by bhjertaas** [comment]
We are also experiencing this error. What is status here?

**Comment by jagadishmurugan** [comment]
Looks like there isn't enough permission for the account to query service dcsvc which leads to an exception in the Get-Service! A quick workaround would be to replace the Get-Service (querying all services) with a call to services of interest - vmcompute, vmss, hvhost... Expect the official fix in Jan/2024 release.

**Comment by jagadishmurugan** [comment]
fixed in 1.4.11.19014 (Jan 2024 release)
